### ISSUE TITLE
2.0.3 desktop file validation test fail

### ISSUE THREAD

**Issue by tim77** [Philip-Scott/Notes-up#389]
## Expected Behavior

Notes-up compiles successfully as previous version.

## Current Behavior

FTBFS.

## Possible Solution

We should fix desktop file.

## Steps to Reproduce (for bugs)
1. Build.
2. Error during tests:
```
+ desktop-file-validate /builddir/build/BUILDROOT/notes-up-2.0.3-1.fc35.x86_64/usr/share/applications/com.github.philip-scott.notes-up.desktop
/builddir/build/BUILDROOT/notes-up-2.0.3-1.fc35.x86_64/usr/share/applications/com.github.philip-scott.notes-up.desktop: error: file contains multiple keys named "Comment[fr]" in group "Desktop Entry"
```

- Notes-Up Version: 2.0.3
- OS Info: Fedora 35

**Comment by Philip-Scott** [comment]
Looks like the generated .desktop file from the translations is adding that extra line. I'll fix it for the next release. Thank you!

```
[Desktop Entry]
Name[fr]=Notes-Up
Name=Notes-Up
Comment[fr]=Votre carnet de notes en Markdown
Comment=Your markdown notebook
Comment[lt]=Jūsų markdown užrašinė
Comment[fr]=Votre carnet de notes markdown
Comment[pt_BR]=O seu caderno em Markdown
Exec=com.github.philip_scott.notes-up %u
Icon=com.github.philip_scott.notes-up
Terminal=false
Type=Application
StartupNotify=true
Categories=Office;WordProcessor;
MimeType=application/x-notesup;
```

**Comment by tim77** [comment]
Thanks for quick fix! 2.0.5 built for Fedora and ready for [testing](https://bodhi.fedoraproject.org/updates/?packages=notes-up).
